tools/src/main/kotlin/org/dashevo/tools/DashPayWalletExtension.kt

```diff
@@ -0,0 +1,27 @@
+package org.dashevo.tools
+
+import org.bitcoinj.wallet.Wallet
+import org.bitcoinj.wallet.WalletExtension
+import org.dashevo.dashpay.BlockchainIdentity
+
+class DashPayWalletExtension : WalletExtension {
```

Comment:
This was created as a means of saving DashPay information (identity, username, contacts, etc) into the wallet protobuf file as raw data.  This class would be responsible for managing this.  This wouldn't necessarily be used in `dash-wallet` but may be useful for command line applications.  

In the case of this PR, we don't need this feature because we do the scripts (not currently included) will be doing everything from scratch.